mmv1/third_party/terraform/services/compute/image.go

```diff
@@ -28,6 +29,7 @@ var (
 	resolveImageFamily                 = regexp.MustCompile(fmt.Sprintf("^(%s)$", resolveImageFamilyRegex))
 	resolveImageImage                  = regexp.MustCompile(fmt.Sprintf("^(%s)$", resolveImageImageRegex))
 	resolveImageLink                   = regexp.MustCompile(fmt.Sprintf("^https://www.googleapis.com/compute/[a-z0-9]+/projects/(%s)/global/images/(%s)", verify.ProjectRegex, resolveImageImageRegex))
```

Comment:
Can you take steps to remove the modification of this global value inside of ResolveImage and use the same function you just introduced there too? Effectively, we want to encapsulate "identify and/or break up an image self link" into a function now that it is sufficiently complicated.

Reply:
For this commit, I took the override out and added a case for `resolveImageUniverseLink`